README.md

```diff
@@ -4,10 +4,11 @@ This is the official repository for all things regarding the ForkDela smart cont
 All contracts are located in the `contracts` folder.
 
 ## Summary
+- [Development](#development)
```

Comment:
Can you add all of the subheading sections under here as well

Reply:
I tried and failed... Could you do that please?